models/Post.php

```diff
@@ -136,13 +139,16 @@ public function beforeSave()
     /**
      * Sets the "url" attribute with a URL to this object
      * @param string $pageName
-     * @param Cms\Classes\Controller $controller
+     * @param Controller $controller
+     * @param string[] $params
```

Comment:
`@param string[]` Should be `@param array` and should include a comment describing expected / supported params.